README.md

```diff
@@ -161,6 +161,55 @@ export const env = setEnv({
 
 > **NOTE**: if you choose to assign `optional` and `required` env vars individually, `setEnv` should only be done _once_ for each or you will overwrite your previously defined values.
 
+### Loading .env files
+
+If you set a path to an `.env` file, `simple-env` will parse the file and import the contents into the environment automatically. These will be available in the `process.env` object.
+
+> If you don't specify a path, `simple-env` **will not** automatically load any `.env` files. This is to avoid a breaking change.
+
+```typescript
+// src/env.ts
+import setEnv from '@americanairlines/simple-env';
+
+export const env = setEnv({
+  required: {
+    nodeEnv: 'NODE_ENV',
+    someRequiredSecret: 'SOME_REQUIRED_SECRET',
+  },
+  options: {
+    envPath: "./.env"
+  }
+});
+```
+Each variable needs to be declared on a separate line.
```

Comment:
Let's add all of this to a new sub-section called `Correct .env Syntax` (or something like it). To keep the README simple, can we add all of that to a [`<details>` tag](https://www.w3schools.com/tags/tag_details.asp)?